osf/migrations/0101_add_view_collectionprovider_to_admin_perm.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+import logging
+
+from django.db import migrations
+from django.db.models import Q
+from django.contrib.auth.models import Group
+from django.contrib.auth.models import Permission
+
+logger = logging.getLogger(__file__)
+
+
+def get_new_read_only_permissions():
+    return Permission.objects.filter(
+        Q(codename='view_collectionprovider')
+    )
+
+
+def get_new_admin_permissions():
+    return Permission.objects.filter(
+        Q(codename='change_collectionprovider') |
+        Q(codename='delete_collectionprovider')
+    )
+
+
+def add_group_permissions(*args):
+
+    # Add permissions for the read only group
+    read_only_group = Group.objects.get(name='read_only')
```

Comment:
I know this wasn't you, so I won't ask for it to be changed, but this naming convention is particularly not good.